### PR TITLE
Keystone fix

### PR DESCRIPTION
**Pull request by PiotrProkop**
Depends on #11 

Added:
- keystone/templates/_helpers.tpl

Changed:
- keystone/templates/db-sync.yaml
- keystone/templates/deployment.yaml
- keystone/templates/init.yaml
- keystone/values.yaml

### CODE REVIEW

File: keystone/values.yaml [hunk 1]
```diff
@@ -1,31 +1,60 @@
...
+
+misc:
+  workers: 8
+  debug: false
+
+dependencies:
+  api:
+    jobs: 
+    - mariadb-seed
+    - keystone-init
```
Comment (DTadrzak) [inline keystone/values.yaml]: add or replace because we need to wait for keystone-db-sync

**Comment by PiotrProkop** [comment]
I resolved conflicts.

**Comment by DTadrzak** [comment]
lgtm, Can some else take a look on that?

**Comment by v1k0d3n** [comment]
@PiotrProkop and @DTadrzak everything lgtm. @alanmeadows I'll have you look this over as well, since you've been so close to the recent helm work.

**Comment by alanmeadows** [comment]
lgtm

**Pull request merged**